src/Symfony/Component/Translation/Catalogue/AbstractOperation.php

```diff
@@ -150,9 +150,13 @@ public function getObsoleteMessages($domain)
     /**
      * {@inheritdoc}
      */
-    public function getResult()
+    public function getResult($domain = null)
```

Comment:
I went with adding setDomains(). but I can still try to update if you want.
Also do you want me to squash the 2 commits ?